src/screens/SUE/SueMapScreen.tsx

```diff
@@ -165,6 +177,17 @@ export const SueMapScreen = ({ navigation, route }: Props) => {
 };
 
 const styles = StyleSheet.create({
+  closeButton: {
+    alignItems: 'center',
+    backgroundColor: colors.darkText + '72',
```

Comment:
how does this work?
what about `backgroundRgba`? or `darkerPrimaryRgba`? or `shadowRgba`? or `lighterPrimaryRgba`?

Reply:
let's say `darkText` is `#333333`. If we add `50` to the end of this color code, the color will be 50% transparent.

The color code we are looking for is `overlayRgba`.